infrastructure/server/variables_provided.tf

```diff
@@ -372,4 +372,10 @@ variable "dev_pr_url_count" {
   description = "Number of pull request redirect urls of Cognito dev clients"
   type = number
   default = 50
-}
\ No newline at end of file
+}
+
+variable "oidc_sso_playground_url" {
+  description = "OIDC SSO Playground for debugging congnito login"
+  type = string
+  default = "https://sso-playground.apps.gold.devops.gov.bc.ca"
+}
```

Comment:
It's not clear in this file how it is organized. Would you please place this variable under line 215 or line 55 section? Thanks!

Reply:
https://github.com/bcgov/nr-forests-access-management/pull/1565/commits/e226e2c58c4f46adcb2424c4fa51bdb6e2611f16 Done